src/DynamoPackages/Package.cs

```diff
@@ -528,15 +529,52 @@ internal void MarkForUninstall(IPreferences prefs)
             {
                 prefs.PackageDirectoriesToUninstall.Add(RootDirectory);
             }
-            RaisePropertyChanged(nameof(LoadState.ScheduledState));
+            RaisePropertyChanged(nameof(LoadState));
         }
 
         internal void UnmarkForUninstall(IPreferences prefs)
         {
             LoadState.ResetScheduledState();
 
             prefs.PackageDirectoriesToUninstall.RemoveAll(x => x.Equals(RootDirectory));
-            RaisePropertyChanged(nameof(LoadState.ScheduledState));
+            RaisePropertyChanged(nameof(LoadState));
+        }
+
+        internal void MarkForLoad(IPreferences prefs, bool schedule = true)
+        {
+            if (BuiltInPackage)
+            {
+                if (schedule)
+                {
+                    LoadState.SetScheduledForLoad();
+                }
+                else
+                {
+                    LoadState.SetAsLoaded();
+                }
+
+                if (prefs.PackageDirectoriesToUninstall.Contains(RootDirectory))
+                {
+                    prefs.PackageDirectoriesToUninstall.Remove(RootDirectory);
```

Comment:
When a user wants to Unload a built-in package (form the UI), the built-in package's Root directory will be added to prefs.PackageDirectoriesToUninstall

Now If the user wants to Load that package, we remove the package;'s rootDIr from prefs.PackageDirectoriesToUninstall so that it will not be set as Unloaded on the next dynamo restart.